### ISSUE TITLE
Non-conformant payload returns 200

### ISSUE THREAD

**Issue by blake-regalia** [Open-MBEE/exec-mms#114]
If I submit a JSON payload to `/project/{project_id}/refs` that does not conform to the Swagger model, I get an HTTP 200 response:
```
{
  "message": "[ERROR]: Unable to get object from JSON request"
}
```

However, this should return HTTP 400 so that developer can debug their program or handle invalid user input appropriately.

**Comment by dlamoris** [comment]
do you have the json that you tried with? some simple errors i tried results in 400 or 500

**Comment by blake-regalia** [comment]
I think it was something like this, where i accidentally used a ref object instead of an array:
```json
{
  "refs": {
    "id": "branchX",
    "name": "Branch X",
    "parentRefId": "master",
    "type": "Branch"
  }
}
```

**Comment by HuiJun** [comment]
Seems to be fixed in #145
